internal/app/app.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/brotigen23/go-url-shortener/internal/config"
+	"github.com/brotigen23/go-url-shortener/internal/server"
+	"github.com/brotigen23/go-url-shortener/internal/storage"
+)
+
+func Run() {
+	config := config.NewConfig()
+	storage := storage.NewStorage()
+
+
+	err := server.Run(config, storage)
+
+
+
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
```

Comment:
[CHANGE REQUEST] Please use `Exit` only in the main package.
https://github.com/uber-go/guide/blob/master/style.md#exit-in-main